backend/app/assets/javascripts/spree/backend/views/quick_switch.js

```diff
@@ -0,0 +1,71 @@
+Spree.Views.QuickSwitch = Backbone.View.extend({
+  events: {
+    "shown.bs.modal": "onShow",
+    "hidden.bs.modal": "onHide",
+    "submit form": "onSubmit"
+  },
+
+  initialize: function() {
+    this.$el = $(this.el);
+    this.$dialog = $("[data-js='quick-switch-dialog']", this.$el);
+    this.$form = $("form", this.$el);
+    this.$input = $("[data-js='quick-switch-input']", this.$el);
+    this.$tooltip = $("[data-js='quick-switch-tooltip']", this.$el);
+		this.render();
+  },
+
+  triggerShortcut: function() {
+    this.onShortcut();
+  },
+
+  onShortcut: function() {
+    this.$el.modal("toggle");
+  },
+
+  onShow: function() {
+    this.$input.focus();
```

Comment:
How about saving the user's current target if they're already using a form element and returning it to them afterwards?

Reply:
👍

Reply:
If they are already in a form element we won't kick them out.